polaris/list.py

```diff
@@ -130,14 +130,15 @@ def main():
         prog='polaris list')
     parser.add_argument('-t', '--test_expr', dest='test_expr',
                         help='A regular expression for a test path name to '
-                             'search for',
+                             'search for.',
                         metavar='TEST')
     parser.add_argument('-n', '--number', dest='number', type=int,
-                        help='The number of the test to list')
+                        help='The number of the test to list.')
     parser.add_argument('--machines', dest='machines', action='store_true',
-                        help='List supported machines (instead of test cases)')
+                        help='List supported machines (instead of test '
+                        'cases).')
```

Comment:
To make the linter happy:
```suggestion
                        help='List supported machines (instead of test '
                             'cases).')
```